soc/st/stm32/stm32h5x/Kconfig.defconfig.stm32h533xx

```diff
@@ -0,0 +1,11 @@
+# STMicroelectronics STM32H503XX MCU
+
+# Copyright (c) 2024 STMicroelectronics
+# SPDX-License-Identifier: Apache-2.0
+
+if SOC_STM32H533XX
+
+config NUM_IRQS
+	default 134
```

Comment:
can you check, RM0481 gives only positions 0 to  132 for the  STM32H523/533xx vector table